src/main/java/com/facebook/openwifirrm/ucentral/informationelement/BssAcAccessDelay.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.informationelement;
+
+import java.util.Objects;
+
+import com.google.gson.JsonObject;
+
+/**
+ * This information element (IE) appears in wifiscan entries. It currently does
+ * not appear in these entries AFAICT. It's called "BSS AC Access Delay" in
```

Comment:
Minor: I am a fan of not including "AFAICT" in comments (informal and potentially some people don't know the meaning), but up to you.